src/nupic/algorithms/KNNClassifier.py

```diff
@@ -401,7 +402,7 @@ def learn(self, inputPattern, inputCategory, partitionId=None, isSparse=0,
 
     if rowID is None:
       rowID = self._iterationIdx
-
+      
```

Comment:
White space not recoup'd after adding printouts to see what was going on or modifying variables to test the different branches. This is totally not intended...